src/component/navbar.js

```diff
@@ -0,0 +1,189 @@
+import React from 'react';
+import Styled from 'styled-components';
+import MediaQuery from 'react-responsive';
+import FontAwesome from 'react-fontawesome';
+
+import smallLogo from '../assets/images/logos/logo.svg';
+
```

Comment:
Two new lines after imports. Separate the imports from the rest of the code with two new lines.